pkg/nodes/list.go

```diff
@@ -60,3 +68,98 @@ func ListExternalIPv4Networks(apiClient *clients.Settings, options v1.ListOption
 
 	return ipV4ExternalAddresses, nil
 }
+
+// WaitForAllNodesAreReady waits for all nodes to be Ready for a time duration up to the timeout.
+func WaitForAllNodesAreReady(apiClient *clients.Settings,
+	options v1.ListOptions,
```

Comment:
Can you move options to the last parameter? We are going to change this to an optional param in a future PR and don't want to break any tests that use it

Reply:
sure